_docs/installation.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Docker Installation
+permalink: /docs/installation/
+layout: docs
+description: How to install Simpl using Docker
+---
+
+## Docker Installation
+
+You will need to have [Docker](https://www.docker.com/) installed.
+
+At the end of this tutorial, you will have successfully installed and played a single-player `Calc` game.
+
+In order to play a game, you will need to install and run `simpl-games-api`, a game model service, and a game UI.
+
+For every game, you will write the game model service, and the game UI. 
+
+For this tutorial, we will use [simpl-calc-model](https://github.com/simplworld/simpl-calc-model) and [simpl-calc-ui](https://github.com/simplworld/simpl-calc-ui)
+
+Clone the simpl-games-api, simpl-calc-model, and simpl-calc-ui repositories:
+
+```bash
+git clone git@github.com:simplworld/simpl-games-api.git
+git clone git@github.com:simplworld/simpl-calc-model.git
+git clone git@github.com:simplworld/simpl-calc-ui.git
+```
+
+The `simpl-games-api` should always be installed first and will run the entire time.
+
+`cd` into the `simpl-games-api` repo and start Docker.
+
+```bash
+cd simpl-games-api
+docker-compose up
+```
+
+Open another terminal window. In that window, use a `make` script to log into the container, which is named `api`.
+
+```bash
+make shell
+```
+
+A `superuser` and `staff user` will need to be created. 
+
```

Comment:
Once you are in the container shell, create your superuser and staff user by running:

```bash
./manage.py simpl_setup
```

Reply:
I added that to the update I will push soon.